src/MICore/JsonLaunchOptions.cs

```diff
@@ -0,0 +1,392 @@
+﻿// © Microsoft. All rights reserved.
+
+using System;
```

Comment:
This is a generated file that I had to hand tweak to get it to work right (removing some items, adding others, etc.